test/common/ModuleTwin.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Shared;
+    using Newtonsoft.Json.Linq;
+
+    public class ModuleTwin
+    {
+        string deviceId;
+        IotHub iotHub;
+        string moduleId;
+
+        public ModuleTwin(string moduleId, string deviceId, IotHub iotHub)
```

Comment:
use EdgeModule type as constructor argument.

Reply:
See my comments about EdgeDevice/EdgeModule. I started out that way, but determined it wasn't necessary.